modules/core/src/stat.cpp

```diff
@@ -2270,8 +2272,12 @@ static bool ipp_minMaxIdx( Mat &src, double* minVal, double* maxVal, int* minIdx
 
             CV_SUPPRESS_DEPRECATED_START
             ippiMinMaxIndxFuncC1 ippFuncC1 =
+#if IPP_VERSION_X100 != 900 // bug in 9.0.0 avx2 optimization
```

Comment:
That's not correct. ICV supports SSE4.2 and AVX2, but skips AVX.
May be there is should be mechanism for disabling only specific optimizations in case of such bugs, but I don't see any simple way to implement it.